metadata/android/en-US/full_description.txt

```diff
@@ -16,3 +16,6 @@ Highlight of some of the features:
 
 Like any other virtual keyboards, it must be enabled in the system settings. Open the System Settings and go to:
 System > Languages & input > On-screen keyboard > Manage on-screen keyboards.
+
+There are no ads, no tracking, no unnecessary permissions!
```

Comment:
Ads and tracking are mentioned on line 13, you might want to remove the first occurrence.
There's a lot of '!' ! I would prefer if one of them is removed.

Reply:
oh, my bad, didn't notice at the end of the line. 
I removed it.